docs/documentation/logs.mdx

```diff
@@ -90,8 +90,22 @@ You can customize which logs are displayed by using the following flags. These c
 
 In addition to console logs, you can review test results in a **native test report**. The report's file path is provided in the test summary, for example:
 
+### Android:
 ```
-📊 Report: file:///Users/user/patrol/dev/e2e_app/build/app/reports/androidTests/connected/index.html
+Path with build mode (debug, release, profile):
```

Comment:
This line suggests that there can be a path without build mode. Maybe "path without flavor"?